lib/tasks/biran_tasks.rake

```diff
@@ -7,12 +7,31 @@ namespace :config do
     Rake::Task['config:generate_with_deps'].invoke
   end
 
+  task :generate2 do
```

Comment:
@javierg  or anyone else - Last question on this here that I need to resolve. Do we want to try the new approach below to running all the tasks even if one fails, so we get all the output and fail the generate task still fails?
Or do we keep the style above and just fail on the first fail?
I don't want to leave this as generate2. I think this new style has some benefits when deploying in that you can see all the failures so you can go once to the server to make any config changes needed to support the code changes versus deploying, seeing an error, fixing the one error, deploying again, fixing another error, etc. 
The other option is to rename this to something like `generate_all` and then we would have both options available. We would just need to convert the apps to use this in the capistrano tasks, so it would mean touching any app we wanted to use the new feature.

Reply:
@javierg I would like to circle back to this. We have several apps now where I could reuse the sidekiq template to spit out multiple versions of the file. We wouldn't have to maintain 3 separate template files for three separate workers.

Any thoughts on this last question about the errors?

Reply:
IMO we should stick with this new format, and call it `generate`, from the use perspective is the same task, with different behavior, but it is because the previous one didn't work well. SO IMO we keep one style (the new one). And yeah we will need to refactor any app were we update biran with this version, even if we keep the name?

Reply:
The only difference is the output. Current style, biran and thus capistrano will error out on the first error. So if you have three templates and three errors, but one in each template setup... you need to fix and run, then fix and run and then fix and run to see all three errors. 
New format will basically spit out the error, but still count the error, loop through all three and show there was an error. If at the end of the tasks list, we had errors, we still return error to the caller. So on a capistrano run, you would see the error from all three template tasks in one shot.
If we change the generate task to use the new format and just remove the old, there would be no changes that I know of needed for capistrano, as it is just the output that would change.
The only other changes needed would be for apps that want to use the new single template multiple file feature. All other apps would continue as they are. 
I would of course test this with a few apps and deploys prior to doing a new release to try to find any other bugs, but this was pretty well tested when I was working on it with the things I could think of to test.

I could leave the current style is something like `generate_legacy` in case there is some issue and we can always remove later

Reply:
OK, That sounds great, leave it as `generate_legacy`, and I don't see why the new style wont be better.